tests/ui/traits/const-traits/auxiliary/staged-api.rs

```diff
@@ -9,6 +9,11 @@
 pub trait MyTrait {
     #[stable(feature = "rust1", since = "1.0.0")]
     fn func();
+
+    #[stable(feature = "rust1", since = "1.0.0")]
+    fn default<T: ~const MyTrait>() {
+        T::func();
```

Comment:
```suggestion
        // This call is const-unstable, but permitted here since we inherit
        // the `rustc_const_unstable` above.
        T::func();
```